frontend/src/metabase/nav/components/CollectionBreadcrumbs/CollectionBreadcrumbs.tsx

```diff
@@ -13,21 +17,48 @@ import {
 
 export interface CollectionBreadcrumbsProps {
   collection?: Collection;
+  onClick?: (collection: CollectionEssentials) => void;
+  baseCollectionId: CollectionId | null;
+}
+
+function getCollectionList({
```

Comment:
Would it be a good idea to unit test this function?

Reply:
yeah that's a good idea :smile: